pkgs/jnigen/lib/src/elements/elements.dart

```diff
@@ -663,16 +795,103 @@ class JavaDocComment implements Element<JavaDocComment> {
   }
 }
 
+List<TypePathStep> _typePathFromString(String? string) {
+  if (string == null) return const [];
+  const innerClass = 46;
+  assert(innerClass == '.'.codeUnitAt(0));
+  const array = 91;
+  assert(array == '['.codeUnitAt(0));
+  const wildcard = 42;
+  assert(wildcard == '*'.codeUnitAt(0));
+  const digit0 = 48;
+  assert(digit0 == '0'.codeUnitAt(0));
+  const digit9 = 57;
+  assert(digit9 == '9'.codeUnitAt(0));
+  const digitEnd = 59;
+  assert(digit9 == ';'.codeUnitAt(0));
```

Comment:
typo. It's a bit odd that your tests didn't catch this. Is this not covered, or do you need to enable asserts somewhere?

Reply:
Oh thanks for noticing. This is dead code.

Reply:
Oh wait, it's not... I'll add a test to make sure the asserts are running.